### ISSUE TITLE
1.21 release date

### ISSUE THREAD

**Issue by quinn9090** [cabaletta/baritone#4421]
I'd like to know about the time you are planning to be done/release version 1.21 for baritone. If you have an estimate I'd like to know.

Thanks

quinn9090

**Comment by XanSchaakt** [comment]
Download the "Artifacts" from the CI and pick out the correct version: https://github.com/cabaletta/baritone/actions/runs/9593797633

**Comment by FlashlightET** [comment]
thats great and all but did you know artifacts can expire

**Comment by GrahamJenkins** [comment]
@XanSchaakt Appreciate the pointer, but the artifacts have expired. As mentioned in #4557 the current build instructions appear to be incomplete. Would it be possible to retrigger a build / archive the artifacts somewhere / update and clarify build instructions?

Thanks!

**Comment by ZacSharp** [comment]
There is a release for Minecraft 1.21.3 now. Just note that it has a bug rendering Litematica integration unusable outside development environments.

There's also newer CI builds if you look at the [list](https://github.com/cabaletta/baritone/actions/workflows/gradle_build.yml?query=branch%3A1.21.3).

**Comment by GrahamJenkins** [comment]
@ZacSharp Apologies for the minor necro here, but you mention that there is a release for 1.21.3, however this issue is requesting 1.21. Upon testing the mentioned release is compatible with 1.21.2 at oldest. 

I'd like to reiterate that the build instructions appear to be lacking a bit.

Thanks!

**Comment by ZacSharp** [comment]
I don't think there will be a release for 1.21.1 anymore. <- Edit: Prediction changed. There will be a release.

The build instructions basically were removed because they caused more misinformation than they helped. What's left is a guide to install java on MacOS and an incomplete mapping from mc version to java version.
I also don't think maintaining them beyond "just gradlew build like for any other mod" is worth the effort. Catching all the various ways in which gradle can fail is hopeless.
(My strategy in such cases is wiping caches and if that doesn't work I got a problem)
